### ISSUE TITLE
azure did not  support streaming?

### ISSUE THREAD

**Issue by FunkyYang** [sashabaranov/go-openai#765]
I use this sdk to connect with azure chat gpt .and set steam true to request.but the result seems a same one createdTime. 
I wonder why?
![image](https://github.com/sashabaranov/go-openai/assets/5975206/47207393-055a-4791-9498-38a4dff88bd5)

**Comment by FunkyYang** [comment]
streaming seems slowly

**Comment by chrbsg** [comment]
I don't think this is a bug. The "created" field is the same for each streamed partial result in Azure, but it's also the same when using OpenAI's servers. The API docs describe "created" as:

```
created
integer

The Unix timestamp (in seconds) of when the chat completion was created.
```

The multiple parts of a streamed response come from a **single** chat completion, which is probably why the "created" field is always the same. It's a timestamp of the completion as a whole, not a timestamp of individual streamed parts. This field is created by data from the remote server, not go-openai, so either way this isn't a go-openai bug.
